### PR TITLE
[#165350427] Penetration testing no longer requires AWS authorisation

### PR DESCRIPTION
**Pull request by AP-Hunt**
What
----
Penetration testing no longer requires AWS authorisation

However, we still want to hear from tenants when they carry out pen tests. This
commit changes our penetration testing guidance to no longer mentioned the need
for AWS authorisation.

How to review
-------------
1. Review the copy
1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.

Who can review
--------------
@tnwhitwell for technical accuracy, @jonathanglassman for the quality of the copy.

### CODE REVIEW

File: source/documentation/guidance/pentest.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
 ## Penetration testing
 
-You must penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform.
+You [must](#responsibility-model) penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform.
```
Comment (jonathanglassman) [inline source/documentation/guidance/pentest.md]: Links need to be descriptive, so I would suggest 

```
[It is your responsibility](#responsibility-model) to penetration test your app...
```

File: source/documentation/guidance/pentest.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
 ## Penetration testing
 
-You must penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform.
+You [must](#responsibility-model) penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform. 
 
-Because GOV.UK PaaS runs on Amazon Web Services (AWS), you need permission from AWS to penetration test your app. The PaaS team will request permission for you because the process requires root credentials for the AWS portal.
-
-Send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) and the PaaS team will share it with AWS.
+GOV.UK PaaS runs on Amazon Web Services (AWS), but there is no need for us to inform AWS when you carry out penetration tests. However, you should still send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) so that we are aware of it and can approve it.
```
Comment (jonathanglassman) [inline source/documentation/guidance/pentest.md]: Perhaps we could shorten this to focus on what the user must do, rather than what they don't need to do:

```
You must send the following information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) so that we are aware of your test, and can approve it.
```

File: source/documentation/guidance/pentest.md [hunk 3]
```diff
@@ -26,12 +23,7 @@ Send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk
 
 You should supply as much information as possible to minimise delays in this process.
 
-AWS responds to permission requests within two business days. The PaaS team will forward the AWS response to you within two further business days of receiving it.
-
-If your request is approved, you will receive an authorisation number. Your penetration test is only authorised for the dates and times you have specified.
-
-If your request is not approved, the PaaS team will work with you and AWS to solve any issues.
```
Comment (jonathanglassman) [inline source/documentation/guidance/pentest.md]: What happens if your request is not approved?

Comment (AP-Hunt) [reply source/documentation/guidance/pentest.md]: The answer to this was we would probably never reject a request. In which case, are you happy that wording below is sufficient, @jonathanglassman?

> so that we are aware of [your pen test request] and can approve it.

Comment (jonathanglassman) [reply source/documentation/guidance/pentest.md]: Sorry, forgot to reply to this! Yes that sounds good, can we say "penetration" rather than "pen" i.e. `so that we are aware of your penetration test request and can approve it.`
